examples/aspnetcore20-mvc/README.md

```diff
@@ -1,27 +1,38 @@
-ASP.NET Core 2.0 MVC Example Application
-====
+# ASP.NET Core 2.0 MVC Example Application
 
-### Requirements
+This example shows how you can use the Bugsnag .NET notifier in a basic ASP.NET CORE 2.0 MVC app. (We also have an [example](https://github.com/bugsnag/bugsnag-dotnet/tree/master/examples/aspnetcore11-mvc) for ASP.NET CORE 1.3)
+
+## Requirements
 
 - Visual Studio 2017
 - .NET Core 2.0 development tools for web
 
-### Setup
+## Setup
+
+Try this out with [your own Bugsnag account](https://app.bugsnag.com/user/new)! You'll be able to see how the errors are reported in the dashboard, how breadcrumbs are left, how errors are grouped and how they relate to the original source. Don't forget to replace the placeholder API token with your own!
+
+To get set up, follow the instructions below.
+
+1. Clone the repo and `cd` into this directory:
+    ```sh
+    git clone https://github.com/bugsnag/bugsnag-dotnet.git
+    cd bugsnag-dotnet/examples/aspnetcore20-mvc
+    ```
+
+1. In the [Startup.cs](Startup.cs) file Replace the `API_KEY` placeholder with your own Bugsnag API key.
 
-In the `Startup.cs` file enter the Bugsnag API key that you want to use with
-this application:
+1. In the [aspnetcore20-mvc.csproj](aspnetcore20-mvc.csproj) file, make sure `Bugsnag.AspNet.Core` is included in your NuGet packages, as well as  `Microsoft.AspNetCore.All`.
```

Comment:
Is this an instruction that we want people to follow?

Reply:
After discussion, came full circle on this.  Yes, will keep instructions on key installation steps, even if they have already been taken care of by the example app.